package.json

```diff
@@ -33,16 +43,25 @@
     "url": "https://github.com/multiformats/js-multihashing/issues"
   },
   "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.4",
     "blakejs": "^1.1.0",
     "js-sha3": "^0.7.0",
     "multihashes": "~0.4.12",
     "webcrypto": "~0.1.1"
   },
   "devDependencies": {
-    "aegir": "^12.3.0",
+    "aegir": "git+https://github.com/ipfs/aegir.git#flow",
+    "babel-preset-flow-node": "^2.0.1",
     "chai": "^4.1.2",
     "dirty-chai": "^2.0.1",
-    "pre-commit": "^1.2.2"
+    "flow-bin": "^0.69.0",
+    "flow-copy-source": "^1.3.0",
+    "lint-staged": "^7.0.2",
+    "pre-commit": "^1.2.2",
+    "rollup.config.flow": "^1.0.0",
```

Comment:
Same as https://github.com/multiformats/js-multihash/pull/47#pullrequestreview-110264460. No need for rollup I guess.